decidim-core/lib/decidim/webpacker/webpack/base.js

```diff
@@ -1,6 +1,43 @@
 /* eslint-disable */
 
-const { webpackConfig, merge } = require("@rails/webpacker")
+const { webpackConfig, merge, config } = require("@rails/webpacker")
```

Comment:
Why not overriding this on custom.js? It's dangerous to change the base.js as it might change when Webpacker is updated

Reply:
Because we need to modify an existing rule, not to add a new one.

If we would add another `.scss/.sass` rule to `custom.js`, that would come after the one that is at `@rails/webpacker`. Therefore, the `@rails/webpacker` rule would get priority and SCSS compilation would fail because it would fail to find `!decidim-style-imports`.

Reply:
Another way to solve this would be to create `@decidim/webpacker` NPM package which would load `@rails/webpacker` and apply that configuration. I left some comments regarding this approach at https://github.com/decidim/decidim/pull/8119#issuecomment-856899999.

Reply:
Ok, I see, thanks for the clarification.